assets/prototype/domain/eventEditor/data/types.ts

```diff
@@ -47,15 +47,15 @@ export interface DatetimesList {
 }
 
 export interface Price extends Entity {
-	amount?: number;
+	amount?: number | string;
```

Comment:
IMHO, its string case should be handled outside of TS types.

Reply:
i agree, I'll be doing another PR for handling conversions between expected entity types and form data types